packages/docs/celo-codebase/protocol/governance.md

```diff
@@ -13,23 +13,23 @@ The change procedure happens in the following phases:
 3.  Referendum
 4.  Execution
 
-{% hint style="info" %}
-**Note:** the timings mentioned in the rest of this section are for the Alfajores Testnet. It is expected that mainnet timings will be much longer to allow for proper proposal review and engagement.
-{% endhint %}
+### Overview of Phases
+
+Each proposal starts on the **Proposal Queue** where it may receive upvotes for promotion to the Approval phase. Proposals are promoted once per day, and must be promoted within 4 weeks. **Approval** lasts 1 days, during which the proposal must be approved by the Approver. Approved proposals are promoted to the Referendum stage. **Referendum** lasts five days, during which owners of Locked Celo vote yes or no on the proposal. Proposals that satisfy the necessary quorum are promoted to the execution phase. **Execution** lasts up to three days, during which anybody may trigger the execution of the proposal. 
```

Comment:
```suggestion
Each proposal starts on the **Proposal Queue** where it may receive upvotes to move forward in the queue relative to other queued proposals.  Three proposals from the top of the queue are dequeued and promoted to the approval stage per day.  Any proposal that remains in the queue for 4 weeks will expire. **Approval** lasts 1 days, during which the proposal must be approved by the Approver. Approved proposals are promoted to the Referendum stage. **Referendum** lasts five days, during which owners of Locked Celo vote yes or no on the proposal. Proposals that satisfy the necessary quorum are promoted to the execution phase. **Execution** lasts up to three days, during which anybody may trigger the execution of the proposal. 
```